core/src/main/java/org/apache/iceberg/metrics/CommitReport.java

```diff
@@ -27,6 +27,8 @@ public interface CommitReport extends MetricsReport {
 
   String tableName();
 
+  long timestampMillis();
```

Comment:
we shouldn't be breaking existing APIs, so this should rather be 
```
@Value.Default
  default long timestampMillis() {
    return System.currentTimeMillis();
  }
```
That way we don't need to add an exception to RevAPI. See also https://iceberg.apache.org/contribute/#adding-new-functionality-without-breaking-apis